src/app/core/database/pouch-database.ts

```diff
@@ -135,6 +141,19 @@ export class PouchDatabase extends Database {
     });
   }
 
+  putAll(objects: any[], options?: PutAllOptions): Promise<any> {
+    if (options?.force) {
+      objects.forEach((obj) => (obj._rev = undefined));
+    }
+    return this._pouchDB.bulkDocs(objects, options).catch((err) => {
+      if (err.status === 409) {
```

Comment:
I guess its not too relevant for the demo data generation but what happens in case only some of the elements fail to be saved (e.g. wrong `_rec`). Besides that I think the call to `resolveConflicts` does not make sense here as this function can only handle single documents.

Reply:
I have implemented a solution that might do the trick. Can you have a look?